sidebars.ts

```diff
@@ -340,6 +340,15 @@ module.exports = {
                 'send-data/hosted-collectors/http-source/troubleshooting',
               ],
             },
+            {
+              type: 'category',
+              label: 'Webhook Sources',
+              collapsible: true,
+              collapsed: true,
+              items: [
+                'send-data/hosted-collectors/webhook-sources/zoom',
```

Comment:
@JV0812 I've fixed the link here, created a `/webhook-sources` folder, and dropped the Zoom.md file there. Can you take a look?